shared/css/style-constants.scss

```diff
@@ -9,3 +9,6 @@ $resize-bar-width: 13px;
 
 // Opacity of elements that are dragged out to be deleted
 $delete-opacity: 0.5;
+
+// Width of pegasus content as of 2017 redesign
+$pegasus-content-width: 970px;
```

Comment:
let's just do `content-width`. The vision is to have this in dashboard as well. Lots of new dashboard components should be using `ContentContainer` which now uses this variable.